tests/unittests/Foundation/NSFileManagerTests.mm

```diff
@@ -219,7 +219,7 @@
 
     NSDictionary* attributes = @{
         NSFileType : NSFileTypeRegular,
-        NSFileImmutable : @YES,
```

Comment:
Also, setting the "file type" of a file after its creation simply shouldn't be possible o_O